### ISSUE TITLE
Which architectures are supported?

### ISSUE THREAD

**Issue by adrelanos** [vmonaco/kloak#24]
I assume it's developed on amd64?

Is there anything platform specific? Any platforms that might not build or are expected to fail building or fail being functional? I am asking for Debian packaging purposes.

Useful to add to readme. These are potential Debian platforms.

amd64
arm64
armel
armhf
hurd-i386
hurd-amd64
i386
i686
kfreebsd-amd64
kfreebsd-i386
mips
mipsel
powerpc
ppc64el
s390x
sparc

**Comment by vmonaco** [comment]
Yes, amd64. I don't think there is anything that is arch specific, but will double check.
